CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright 2017 The Abseil Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+cmake_minimum_required(VERSION 2.8.12)
+project(absl)
+
+# config / options
+option(ABSL_ENABLE_PIC "Enable / disable position-independent code generation" OFF)
+set(ABSL_STD_CXX_FLAG "-std=c++11" CACHE STRING "c++ std flag (default: c++11)")
+
+# enable ctest
+enable_testing()
+
+list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake)
+
+# use usual unix path 
+include(GNUInstallDirs)
+include(ReleaseDebugAutoFlags)
+
+# pthread
+find_package(Threads REQUIRED)
+
+# check for GTest
+find_package(GTest REQUIRED)
+find_package(GMock REQUIRED)
+include_directories(${GTEST_INCLUDE_DIRS} ${GMOCK_INCLUDE_DIRS})
```

Comment:
Nit: In bazel, each cc_test links to GTest explicitely.
This could be done in CMake as well, `include_directories` is usually recognized as a bad form.